### DISCUSSION TITLE
میکرو سرویس

### DISCUSSION THREAD

**Discussion by hamedrajabpour1** [AliMD/alwatr#255] [Educational]
با سلام و عرض احترام

در صورت امکان میکروسرویس نوشتن را به صورت فنی توضیح مختصری ارائه بدید

پیشنهاد بنده پیاده سازی یک پروژه کوچک مثلا وبلاگ ساده با امکان لاگین کردن ادمین جهت پست گذاشتن و فراخوانی پست ها و... که در یک جلسه کوتاه قابل پیاده سازی باشد.

**Comment by MM25Zamanian** [comment]
سال هاست پیشنهاد و خواسته من و چندین نفر دیگه هم همینه
اما تا الان فقط با توجه به پروژه های کوچکی که در الوتر بوده فقط یه جزئیاتی از بحث میکرو سرویس رو یاد گرفتم

دوست داشتین میتونم در اختیارتون بزارم
اما خیلی کلی هستن و اطلاعات خاصی بهتون اضافه نمیکنن
